packages/cycle-scripts-one-fits-all/README.md

```diff
@@ -5,11 +5,12 @@
 ## Language
 
 ES6 or Typescript 2.1 configured with:
-* [ES2015 preset](https://babeljs.io/docs/plugins/preset-es2015/)
-
+* TypeScript targets ES6 with typings for [DOM,ES6,DOM.Iterable,ScriptHost](http://www.typescriptlang.org/docs/handbook/compiler-options.html)
```

Comment:
Yeah, I see. And I was so careful. Yo need to take both together to see entire picture :(